python/lsst/analysis/drp/calcFunctors.py

```diff
@@ -13,7 +13,7 @@ def columns(self):
         pass
 
```

Comment:
You'll need to return `[self.column, f'{self.column}Err']` here, since both are needed in the calculation.

Reply:
I think that might exist on some uncommitted things of mine, I will check. I didn't know we were looking at these specifically yet, so sorry for being behind.

Reply:
I'm not really looking at things in detail generally; I had just cloned the updates and noticed this, so figured I'd mention it.